src/resources/how_to/how_to_integrate_gas_station.md

```diff
@@ -0,0 +1,313 @@
+# How to Integrate Gas Station for Sponsored Transactions
+
+Gas Station lets your dApp pay gas fees for users, removing the "first-buy-APT" hurdle and smoothing onboarding. Use it when you want newcomers to transact with empty wallets or need precise control over which functions are subsidized.
+
+## Prerequisites
+
+Make sure you have published your Move module and have the module address.
+
+## Backend Setup in Aptos Build
+
+1. Go to [Aptos Build](https://build.aptoslabs.com/) and create a new project.
+
+2. Choose "Gas Station" as the project type.
+
+3. Configure your Gas Station:
+   - Name the project (3-32 chars, lowercase, numbers, `_` or `-`)
+   - Select network (Mainnet or Testnet)
+   - Paste your module address and select module name
+   - Select the function(s) to subsidize
+   - Set gas limits (default: 3-50 per transaction, 100 OCTA gas price)
+
+4. Create the Gas Station and copy the auto-generated API key (`aptoslabs_xxx...`).
+
+**Note**: For enhanced rate limits and better performance when making SDK calls to read blockchain data, consider also setting up Full Node API keys. See [how_to_manage_api_keys_as_project_resources.md](./how_to_manage_api_keys_as_project_resources.md) for details.
+
+## Frontend Integration (Plugin Approach - Recommended)
+
+This is the preferred method recommended by the Aptos team. It integrates seamlessly with the wallet adapter and requires minimal setup.
+
+1. **Install required dependencies**:
+
+```bash
+npm install @aptos-labs/ts-sdk@^3.1.1 @aptos-labs/gas-station-client@^1.1.1 @aptos-labs/wallet-adapter-core@^6.0.0 @aptos-labs/wallet-adapter-react@^7.0.0
+```
+
+2. **Configure Gas Station client and Aptos SDK**:
+
+```tsx
+// src/lib/gasStation.ts
+import { AptosConfig, Aptos, Network } from "@aptos-labs/ts-sdk";
+import { createGasStationClient } from "@aptos-labs/gas-station-client";
+
+const network = process.env.NEXT_PUBLIC_NETWORK === 'mainnet'
+  ? Network.MAINNET
+  : Network.TESTNET;
+
+// Create Gas Station client
+const gasStationClient = createGasStationClient({
+  network,
+  apiKey: process.env.NEXT_PUBLIC_GAS_STATION_API_KEY!,
```

Comment:
The agent would implement it as NEXT or VITE or in whatever format the dapp requires

```suggestion
  apiKey: GAS_STATION_API_KEY,
```